### PR TITLE
Remove Travis

### PR DESCRIPTION
**Pull request by gaborbernat**
@kpfleming you'll need to remove the Travis check requirement for this.

### CODE REVIEW

**Comment by kpfleming** [comment]
The 'required' check has been disabled. I haven't stopped Travis-CI from making builds yet, but that can be done as soon as you are comfortable that the Actions workflows have replaced it.

**Comment by gaborbernat** [comment]
@erikseulean can you approve, I think Github has with this entirely replaced Travis.

**Comment by kpfleming** [comment]
There's going to be a small ongoing complication with this: using Actions you are able to define the names of the workflows and jobs, but you aren't able to control the jobs that are marked 'required' in the branch protection settings (because that is handled via automation). Once we setup branch protection to 'require' one or more Actions workflows to be successful for a PR to be mergeable, if you change the workflow or job names then the PRs will stop being mergeable because they'll be looking for the old names.

In other words: be certain that you are happy with your workflow/job names now :-)

**Comment by gaborbernat** [comment]
On my side I'm happy.

**Comment by erikseulean** [comment]
To be fair, I don't see those changing very often so I'm happy with them as they are as well.

**Comment by kpfleming** [comment]
OK, if you look at the list of 'checks passed' on this PR, which ones do you want marked as 'required' for a PR to be mergeable?

**Comment by erikseulean** [comment]
I'd say `lint`, `3.8` and `3.7` . I'm not sure about the `check docs` tbh. @gaborbernat ?

**Comment by erikseulean** [comment]
I'd say `lint`, `3.8` and `3.7` . I'm not sure about the `check docs` tbh. @gaborbernat ?

**Comment by gaborbernat** [comment]
lint and 3.8 for me it's enough

**Comment by kpfleming** [comment]
Done.
